src/platforms/kotlin/config.yml

```diff
@@ -0,0 +1,10 @@
+title: Kotlin
+caseStyle: camelCase
+supportLevel: production
+sdk: sentry.kotlin
+fallbackPlatform: java
+categories:
+  - mobile
```

Comment:
could also be desktop and server, as its pretty much as the Java one

Reply:
I'll add desktop and server.